widgets/buttonWidget.js

```diff
@@ -35,7 +35,7 @@ class ButtonWidget extends Widget {
     this.tpl = `
       <div class="widget" id="widget_{0}">
         <div class="widget-title">
-          <div class="widget-name">{1}</div>
+          <div class="widget-name"><span class="uppercase bold">{1}</span> - <span>#{3}</span></div>
```

Comment:
The # results in headers like this:
ALL - #/demo/#
This is not correct, because only /demo/# is used.